### ISSUE TITLE
Is it possible to support k8s ingress annotations through istio ingress gateway ?

### ISSUE THREAD

**Issue by shuxingzfw** [istio/istio#43849]
We plan to use istio ingress controller to manage k8s ingress。Does istio have plans to support  k8s ingress annotations。

This is the k8s ingress configuration.

`kind: Ingress
metadata:
  name: foo.bar.com
  namespace: default
  annotations:
    nginx.ingress.kubernetes.io/rewrite-target: /$2
spec:
  rules:
  - host: foo.bar.com
    http:
      paths:
      - path: /svc(/|$)(.*)
        backend:
          serviceName: web1-service
          servicePort: 80`

istio will ignore annotation `nginx.ingress.kubernetes.io/rewrite-target`

**Comment by howardjohn** [comment]
No, this is intentionally not supported in favor of Istio Gateway and gateway-api.org
